src/TestProblem/Hydro/ClusterMerger/Init_TestProb_ClusterMerger.cpp

```diff
@@ -786,3 +976,129 @@ void AddNewField_ClusterMerger()
 }
 
 #endif
+
+
+// Restart only: initialize the BH variables (position, velocity, mass) from particles labelled with PTYPE_CEN 
+void Init_User_ClusterMerger()
+{
+   if ( OPT__INIT == INIT_BY_RESTART ){
+
+      if ( OPT__RESTART_RESET ) {
+         printf("Error! OPT__RESTART_RESET should be disabled.\n");
+         return;
+      }    
+
+      const char FileName[] = "BH_variable.bin";
```

Comment:
In a future version it would be nice to store this information in the HDF5 file instead.